File: src/main/java/ru/berdnikov/school2024/dto/OrderDTO.java
package ru.berdnikov.school2024.dto;

import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import ru.berdnikov.school2024.utils.DateTime;

import java.math.BigDecimal;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class OrderDTO {
    private String user_id;

    @JsonDeserialize(using = DateTime.class)
    private DateTime ordered_at;

    private String status;
    private BigDecimal total;
}

File: src/School2024.ServicesForTestTask/Models/DTOs/OrderDTO.cs
namespace School2024.ServicesForTestTask.Models;

using School2024.Application;

// <summary>
// DTO сущность для базовой сущность Заказа
// </summary>

public class OrderDTO : IDTO
{
	public Guid UserId {get; set;}

	public DateTime OrderedAt {get; set;}

	public string Status {get; set;}

	public string Total {get; set;}

}